src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -1,36 +1,172 @@
-import { Grid, Typography } from '@mui/material';
-import { Button, DialogBox } from '@terraware/web-components';
+import { Box, Grid } from '@mui/material';
+import { Button, DatePicker, DialogBox, Select, SelectT } from '@terraware/web-components';
 import { Accession2 } from 'src/api/accessions2/accession';
+import { ViabilityTestPostRequest } from 'src/api/accessions2/viabilityTest';
 import strings from 'src/strings';
+import useForm from 'src/utils/useForm';
+import { Dropdown } from '@terraware/web-components';
+import { SEED_TYPES, SUBSTRATES, TEST_METHODS, TREATMENTS } from 'src/types/Accession';
+import { OrganizationUser, User } from 'src/types/User';
+import { ServerOrganization } from 'src/types/Organization';
+import { useEffect, useState } from 'react';
+import { getOrganizationUsers } from 'src/api/organization/organization';
+import { isContributor } from 'src/utils/organization';
+import TextField from '@terraware/web-components/components/Textfield/Textfield';
+import { postViabilityTest } from 'src/api/accessions2/viabilityTest';
+import useSnackbar from 'src/utils/useSnackbar';
+import { renderUser } from '../withdraw/WithdrawModal';
```

Comment:
we could move renderUser to a utils file? even if within accession2/utils directory that's fine